### PR TITLE
Chinese Translation Improvement

### PR DESCRIPTION
**Pull request by LesterLian**
Improve Chinese Translation by:

- Add new strings since last translation
- Fix error
- Try to maintain consistency of stranslation and Chinese punctuation usage
- Remove unnecessary spaces

### CODE REVIEW

File: MinecraftClient/Resources/lang/zh-CHS.ini [hunk 1]
```diff
@@ -355,147 +361,147 @@ cmd.useitem.use=使用了一个物品
...
 # AutoAttack
 bot.autoAttack.mode=未知的攻击模式：{0}，使用单一模式作为默认值。
-bot.autoAttack.priority=未知优先级：{0}，使用距离优先级作为默认值。
+bot.autoAttack.priority=未知优先模式：{0}，使用距离优先作为默认值。
 bot.autoAttack.invalidcooldown=攻击冷却时间值不能小于0，已使用自动作为默认值
 
 # AutoCraft
-bot.autoCraft.cmd=自动创建ChatBot命令
-bot.autoCraft.alias=自动创建ChatBot命令别名
+bot.autoCraft.cmd=自动制作ChatBot命令 # TOCHECK
```
Comment (LesterLian) [inline MinecraftClient/Resources/lang/zh-CHS.ini:368]: One thing I want to confirm before finalizing:  
In similar strings of other bots, the name of the bot is included here, but 'Auto-crafting' is not the name of this bot. I'm not sure if I should translate 'Auto-crafting' or leave it as is.

**Comment by ORelio** [comment]
Can't review Chinese translation but hopefully @ReinforceZwei should be able to handle your pull request 🙂

File: MinecraftClient/Resources/lang/zh-CHS.ini (hunk 1 above)
Comment (ReinforceZwei) [reply MinecraftClient/Resources/lang/zh-CHS.ini:368]: `Auto-crafting` mean automatically craft something. It is not the name of the bot so you may translate it into Chinese.

**Review by ReinforceZwei — APPROVED** [review]

**Comment by ReinforceZwei** [comment]
Is this finished?

**Comment by LesterLian** [comment]
> Is this finished?

Yes, I think so.

**Comment by ORelio** [comment]
Thanks @ReinforceZwei for reviewing this 👍

**Comment by ReinforceZwei** [comment]
@ORelio The build job is failing with missing reference error. Could you take a look at it?

**Pull request merged**